src/Commands/Command.php

```diff
@@ -252,6 +259,16 @@ public function getName()
         return $this->name;
     }
 
+    /*
+     * Get Show in Help
+     *
+     * @return bool
+     */
+     public function getShowInHelp()
```

Comment:
Instead of `getShowInHelp`, just use `showInHelp`. Similar to `isEnabled`, it makes it much easier to "read" the code.

Compare:
```php
$command->getShowInHelp();
```
to
```php
$command->showInHelp();
```
See what I mean? 😃